net/tmate/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+tmate -V | grep "${2#*v}"
```

Comment:
```suggestion
tmate -V | grep "$PKG_VERSION"
```
Instead of grepping via this expression you could also use an exported variable.